File: RockitCompiler/benchmarks/bench_list_build.go
package main

import "fmt"

func main() {
	n := 500000
	var arr []int
	for i := 0; i < n; i++ {
		arr = append(arr, i)
	}
	sum := 0
	for _, v := range arr {
		sum += v
	}
	fmt.Println(sum)
}

File: RockitCompiler/benchmarks/bench_list_build.js
const n = 500000;
const arr = [];
for (let i = 0; i < n; i++) {
    arr.push(i);
}
let sum = 0;
for (let i = 0; i < arr.length; i++) {
    sum += arr[i];
}
console.log(sum);
